packages/performance/src/services/remote_config_service.ts

```diff
@@ -75,12 +67,12 @@ const FIS_AUTH_PREFIX = 'FIREBASE_INSTALLATIONS_AUTH';
 export function getConfig(iid: string): Promise<void> {
   const config = getStoredConfig();
   if (config) {
-    processConfig(iid, config);
+    processConfig(config);
     return Promise.resolve();
   }
 
   return getRemoteConfig(iid)
-    .then(config => processConfig(iid, config))
+    .then(config => processConfig(config))
```

Comment:
This can be abbreviated as
.then(processConfig)

Reply:
Done.